ner_telhub/model/data_models.py

```diff
@@ -385,3 +391,15 @@ def exportCSV(*args, **kwargs) -> None:
                     writer.writerow(
                         [str_time, id, desc, data[1], DATA_IDS[id]["units"]])
         message_signal.emit("Finished CSV export")
+
```

Comment:
Rather than do this across the entire model manager, can you implement it for each data model? This would involve adding a check to `addData()` in the data model class for if the size is greater than the max live data points, and then calling deleteFirstData if so

We would also have to have a checkbox maybe on the top toolbar in the graph dashboard that allows the user to enter this 'live mode'.